Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>123
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 7a81b7666ffaefdfb2d12ae47c4c1345c0e54227)
+++ b/main.py	(date 1768501757457)
@@ -1,1 +1,1 @@
-123
\ No newline at end of file
+qwerty = "qwerty"
\ No newline at end of file
